src/prelaunch_data_generator/cl_validator_keystores/cl_validator_keystore_generator.star

```diff
@@ -121,5 +123,116 @@ def generate_cl_validator_keystores(
 	plan.remove_service(service_name)
 	return result
 
+# this is like above but runs things in parallel - for large networks that run on k8s or gigantic dockers
+def generate_cl_valdiator_keystores_in_parallel(
+	plan,
+	mnemonic,
+	participants,
+	num_validators_per_node):
+
+	service_names = []
+
+	for idx in range(0, len(participants)):
+		service_name = prelaunch_data_generator_launcher.launch_prelaunch_data_generator(
+			plan,
+			{},
+			"cl-validator-keystore-" + str(idx),
+		)
+		service_names.append(service_name)
+
+	all_output_dirpaths = []
+	all_generation_commands = []
+
+	for idx, participant in enumerate(participants):
+		output_dirpath = NODE_KEYSTORES_OUTPUT_DIRPATH_FORMAT_STR.format(idx)
+
+		start_index = idx * num_validators_per_node
+		stop_index = (idx+1) * num_validators_per_node
+		generation_finished_filepath = KEYSTORE_GENERATOIN_FINISHED_FILEPATH_FORMAT.format(idx)
```

Comment:
can we change this to: 

`KEYSTORE_GENERATION_FINISHED_FILEPATH_FORMAT = "/tmp/keystores_generated-{0}-{1}"`
`KEYSTORE_GENERATION_FINISHED_FILEPATH_FORMAT.format(start_index,stop_index)`